include/lcp-catlist.php

```diff
@@ -338,7 +338,7 @@ public function get_instance(){
   public function get_date_to_show($single){
     if ($this->params['date'] == 'yes'):
       //by Verex, great idea!
-      return get_the_time($this->params['dateformat'], $single);
+      return ' ' . get_the_time($this->params['dateformat'], $single);
```

Comment:
Well, probably not a good idea to hardcode it like this, I think I wanted to make the template cleaner but that's probably not worth it.